debian/yaru-theme-unity.install

```diff
@@ -0,0 +1,2 @@
+usr/share/themes/Yaru/unity
```

Comment:
For futur-proofing (snow theme? ;)), I would change this to `usr/share/themes/Yaru*/unity` and removing the following line. WDYT?

Reply:
smart ;)